pages/join.html

```diff
@@ -5,46 +5,29 @@
 home: false
 ---
 
-<h2>Join content goes here.</h2>
-
-<div class="contentContainer">
-  <div class="cardsContainer">
-    <div class="padded-content cards">
-      <div class="flexRow">
-        <div class="col-md-auto image-right-card-text">
-          <div class="card-info">
-            <h4 class="cardTitle">LOREM IPSUM</h4>
-            <h2 class="cardText">Who leads our teams?</h2>
-            <p class="cardText">Project and Product Managers lead our teams. They help define the roadmap and create tickets in our kanban board. We'll try to define MVPs and create tasks to get there.</p>
-          </div>
-        </div>
-        <div class="col-md-auto card-img-container">
-          <img src="/images/proxy-img.png" alt=""/>
-        </div>
+<div class="volunteer-expectations">
+  <h2 class="volunteer-title">What are volunteer <br>expectations?</h2>
```

Comment:
I think this styles fine without the break tag. We should leave it out.

Reply:
removed <br>